controllers/managedocs_controller_test.go

```diff
@@ -322,5 +325,213 @@ var _ = Describe("ManagedOCS controller", func() {
 				}, duration, interval).Should(BeTrue())
 			})
 		})
+
+		When("there is an add-on delete comfigmap", func() {
+			ctx := context.Background()
+
+			managedOCS := &v1.ManagedOCS{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      ManagedOCSName,
+					Namespace: TestNamespace,
+				},
+			}
+			sc := &ocsv1.StorageCluster{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      storageClusterName,
+					Namespace: TestNamespace,
+				},
+			}
+
+			storageClassRbdName := "ocs-storagecluster-ceph-rbd"
+			storageClassCephFsName := "ocs-storagecluster-cephfs"
+
+			pvc1 := &corev1.PersistentVolumeClaim{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "test-pvc1",
+					Namespace: TestNamespace,
+				},
+				Spec: corev1.PersistentVolumeClaimSpec{
+					AccessModes: []corev1.PersistentVolumeAccessMode{corev1.ReadWriteOnce},
+					Resources: corev1.ResourceRequirements{
+						Requests: corev1.ResourceList{
+							corev1.ResourceStorage: resource.MustParse("1Gi"),
+						},
+					},
+					StorageClassName: &storageClassRbdName,
+				},
+			}
+			pvc2 := &corev1.PersistentVolumeClaim{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "test-pvc2",
+					Namespace: TestNamespace,
```

Comment:
There needs to be a test that covers when the PVC is outside of the namespace that the operator is in.

Reply:
Modified existing tests to create one of the PVC outside the operator namespace

Reply:
ty